string act
string name
string drink